### PR TITLE
Add BlueWallet

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by pedro-roberto** [comment]
Is this for mainnet? 
https://itunes.apple.com/us/app/bluewallet-bitcoin-wallet/id1376878040?l=ru&ls=1&mt=8

**Comment by Overtorment** [comment]
Will be, not yet merged to master and released to itunes (90% done)
Bitcoin wallet itself is functional tho

File: wallets.json [hunk 1]
```diff
@@ -74,5 +74,11 @@
...
    "requires_node":"yes",
     "open_source":"https://github.com/LN-Zap/zap-iOS"
-  }
+  },
+  {
+    "name":"BlueWallet iOS",
+    "page":"https://bluewallet.io/",
+    "requires_node":"no",
+    "open_source":"https://github.com/Overtorment/BlueWallet"
+  },
```
Comment (pedro-roberto) [inline wallets.json]: remove comma

Comment (Overtorment) [reply wallets.json]: fixed

**Comment by Overtorment** [comment]
@pedro-roberto  not deployed yet..? 
http://lightningnetworkstores.com/wallets.json

**Comment by lightningnetworkstores** [comment]
Had noticed this, seems like I forgot to turn on the import for this file
in the production server, will fixbit this weekend...

**Comment by pedro-roberto** [comment]
it's there now, sorry for the delay.

**Comment by Overtorment** [comment]
cool thanks

**Pull request merged**